docs/modules/ROOT/pages/how-tos/testing/integration/index.adoc

```diff
@@ -1 +1,17 @@
-= Integration tests
\ No newline at end of file
+= Integration tests
+
+This document covers the integration tests that {ProductName} triggers after component build pipelines. 
+
+Integration tests ensure that all built components conform to some set of requirements. xref:how-tos/testing/integration/adding.adoc[Adding an integration test] is as simple as providing a reference to a git repository and the path to the Tekton pipeline that defines the test. After the required integration tests pass, all of the included components are promoted to the application's Global Candidate List indicating their suitability to be included in the next snapshots and releases.
```

Comment:
Caught another reference to Global Candidate List being updated after testing. I think that the entire sentence can be removed, e.g.:
```suggestion
Integration tests ensure that all built components conform to some set of requirements. xref:how-tos/testing/integration/adding.adoc[Adding an integration test] is as simple as providing a reference to a git repository and the path to the Tekton pipeline that defines the test.
```